DDDEastAnglia.Tests/DateExtensions.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+
+namespace DDDEastAnglia.Tests
+{
+    public static class DateExtensions
+    {
+        public static DateTime January(this int day, int year)
```

Comment:
@adrianbanks  Doesn't look like it supports the same syntax to me.

`In.FebruaryOf(2009) // Returns 1st of February of 2009`

vs

`1.February(2009)`

doesn't look like you can do 

`2.February(2009)`